### PR TITLE
Public Key Cryptography: Cover behaviors in eyes tests

### PR DESCRIPTION
**Pull request by islemaster**
Makes publicKeyCryptography/eyes.feature actually interact with the widgets.  Adds some class names to elements on the widgets to make them easier to find during tests.

### CODE REVIEW

File: apps/src/publicKeyCryptography/Alice.jsx [hunk 1]
```diff
@@ -118,7 +118,11 @@ const Alice = React.createClass({
               onChange={this.onPrivateKeyChange}
               disabled={disabled}
             />
-            <div>Your computed <KeywordPublicKey/> is <IntegerField color={COLORS.publicKey} value={publicKey}/></div>
+            <div>
+              Your computed <KeywordPublicKey/>
+              {' is '}
```
Comment (breville) [inline apps/src/publicKeyCryptography/Alice.jsx:123]: neat trick

File: apps/src/publicKeyCryptography/CollapsiblePanel.jsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const CollapsiblePanel = React.createClass({
     };
 
     return (
-      <div>
+      <div className={`panel-${this.props.title.toLowerCase().replace(/\s+/g, '-')}`}>
```
Comment (breville) [inline apps/src/publicKeyCryptography/CollapsiblePanel.jsx:52]: getting complex enough to hoist into a variable?

**Comment by breville** [comment]
lgtm

File: dashboard/test/ui/step_definitions/publicKeyCryptography_steps.rb [hunk 3]
```diff
@@ -10,3 +10,88 @@
...
+end
+
+When /^I calculate Bob's public number$/ do
+  @browser.find_element(:css, '.panel-bob button').click
+  sleep 6
+end
+
+When /^Alice decodes Bob's secret number$/ do
+  @browser.find_element(:css, '.panel-alice button').click
+  sleep 6
```
Comment (asherkach) [inline dashboard/test/ui/step_definitions/publicKeyCryptography_steps.rb:73]: Is it really necessary to sleep for six seconds in these tests?

Comment (islemaster) [reply dashboard/test/ui/step_definitions/publicKeyCryptography_steps.rb:73]: These are waiting on an animation to complete.  If you'd like to speed these tests up, we'll need to expose a JavaScript hook in the PKC widget that allows us to accelerate this animation or skip it entirely.

**Pull request merged**